django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -58,10 +59,11 @@
         // add controls to the editor header
         advanced.find('.jsoneditor-menu')
             .append($('<a href="javascript:;" class="jsoneditor-poweredBy"><img class="icon" src="/static/admin/img/icon-deletelink.svg" /> back to normal mode</a>').click(function(){
-                toggleFullScreen();
+                if(isValidJson(editor))
+                    toggleFullScreen();
                 advanced.parents('.field-config').find('.normal-mode').click();
             }))
-            .append(advanced.parents('.field-config').find('#netjsonconfig-hint').clone(true))
+            .append(advanced.parents('.field-config').find('#netjsonconfig-hint').clone(true).attr('id','netjsonconfig-hint-1'))
```

Comment:
the line is becoming a bit too long. Could you write something like the following?

```javascript
.append(advanced.parents('.field-config').find('#netjsonconfig-hint')
.clone(true)
.attr('id','netjsonconfig-hint-1'))
```